apps/breethe/lib/breethe/data/data.ex

```diff
@@ -34,7 +34,9 @@ defmodule Breethe.Data do
   end
 
   def create_location(params) do
-    case find_location(params) do
+    %{identifier: identifier} = params
+
+    case find_location_by_identifier(identifier) do
```

Comment:
You could also do `params.identifier` instead of the pattern match.

Reply:
ah, right 🤦‍♂️ - fixed